src/layout/layout.rs

```diff
@@ -357,10 +357,7 @@ impl Layout {
     /// equal chunks or to not distribute extra space at all (which is the default used for laying
     /// out the columns for [`Table`] widgets).
     ///
-    /// Note: If you're using this feature please help us come up with a good name. See [Issue
-    /// #536](https://github.com/ratatui-org/ratatui/issues/536) for more information.
-    ///
-    /// [`Table`]: crate::widgets::Table
+    /// This function exists for backwards compatibility reasons. Use [`Layout::flex`] instead.
```

Comment:
Note the equivalent values here or in the doc string

Reply:
I added a docstring with the mapping.